src/pages/about-carbon-for-ibm.com/what-we-provide.mdx

```diff
@@ -1,51 +1,116 @@
 ---
 title: What we provide
-description: Carbon for IBM.com is the design system for IBM.com's web
-  experiences. The system consists of working code, design tools, resources for page owners, designers, and developers creating pages for the IBM.com website.
 ---
 
+import { Link } from 'gatsby';
+
 <PageDescription>
 
-Carbon for IBM.com is the open source design system for IBM.com's web experiences. The system consists of working code, design tools, and resources for page owners, designers, and developers creating pages for the IBM.com website.
+Carbon for IBM.com provides guidance and resources for designers and developers
+creating pages and applications on IBM.com and other IBM domains. The team is
+here to help you on your path to adoption.
 
 </PageDescription>
 
+<InlineNotification>
+
+**Note:** Carbon for IBM.com is the strategic design system for IBM web
+experiences. Any application using older systems like Northstar (v18), v19, or
+even earlier versions should make plans to adopt Carbon for IBM.com by the end
+of 2021. After all applications have migrated, plans and dates will be announced
+for sunsetting the older systems. Reach out to us in our IBM Slack channel at
+[#carbon-for-ibm-dotcom](https://ibm-studios.slack.com/archives/C2PLX8GQ6) with
+any questions.
+
+</InlineNotification>
+
 <AnchorLinks>
 
-<AnchorLink>About Carbon for IBM.com</AnchorLink>
-<AnchorLink>About Carbon</AnchorLink>
-<AnchorLink>Key differences</AnchorLink>
-<AnchorLink>Questions</AnchorLink>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>What is included in Carbon for IBM.com?</AnchorLink>
+<AnchorLink>Choosing the right path to adoption</AnchorLink>
+<AnchorLink>Resources</AnchorLink>
 
 </AnchorLinks>
 
-## About Carbon for IBM.com
+## Overview
+
+IBM.com is a place for people to learn about, try, and buy IBM products and
+services. Visitors to IBM.com deserve to have an efficient and delightful
+experience while on their journey.
+
+With that goal in mind, Carbon for IBM.com provides guidance and resources that
+are built on the foundations of the Carbon Design System and the IBM Design
+Language. These resources ensure that creators of experiences for IBM.com are
+building consistent experiences across the entire site.
+
+## What is included in Carbon for IBM.com?
+
+Carbon for IBM.com is focused on supporting IBM.com applications and websites.
+It provides features and components relevant to IBM.com.
+
+- Components usage and code (e.g. Masthead and Footer)
+- Layout patterns usage and code (e.g. Lead Space and Table of Contents)
+- Services and utilities (e.g. required Tealium)
+- Implementation guidelines
+- Sketch design kits
+- Links to sandbox and code resources
+- Metrics
+- Accessibility tested and compliant
+- Optimized code
```

Comment:
Seems to be an unnecessary mention. All code should be provided in the best shape possible. IMO. Would welcome a dev perspective.